Bf16* fp4 gemm with bias and swiglu (#2835)
ROCm/composable_kernel: example/ck_tile/41_bf16_mxfp4_bscale_gemm_act/README.md

# GEMM Matrix Multiplication

> This folder contains example for GEMM using ck_tile tile-programming implementation. Currently, it only supports the basic feature of the CK Tile GEMM, but creates the placeholders for the future support on different GEMM pipeline and different GEMM modules. In the near future, we will gradually migrate all the GEMM features from old CK to CK Tile.

Reviewer: ```suggestion
This folder contains GEMM examples that use ck_tile tile-programming implementations. Currently, only the basic CK Tile GEMM features are supported, but placeholders for future support on different GEMM pipeline and different GEMM modules are included. In the near future, we will gradually migrate all the GEMM features from old CK to CK Tile.
```

## build
```
# in the root of ck_tile
mkdir build && cd build
# you can replace <arch> with the appropriate architecture (for example gfx90a or gfx942) or leave it blank
sh ../script/cmake-ck-dev.sh  ../ <arch>
# The pipeline method on the gemm calculation
make tile_exampl_bf16_mxfp4_gemm_act -j

```

> This will result in an executable `build/bin/make tile_exampl_bf16_mxfp4_gemm_act -j

Reviewer: This line is ambiguous.  The make command is already above, so in this line you'd expect information about where the binary file has been written to.  Instead, there's a reiteration of the make command, but using a different path to make.
